library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/CompanyDoctrineRepository.php

```diff
@@ -59,6 +59,32 @@ function ($row): int {
         );
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function findDomainIdsByBrandId(int $id): array
+    {
+        $qb = $this->createQueryBuilder('self');
+        $expression = $qb->expr();
+
+        $qb
+            ->select('IDENTITY(self.domain) as domain')
+            ->where(
+                $expression->eq('self.brand', $id)
+            );
+
+        $result = $qb
+            ->getQuery()
+            ->getScalarResult();
+
+        return array_map(
+            function ($row): int {
+                return (int) $row['domain'];
+            },
+            $result
+        );
+    }
```

Comment:
This makes me think that Domain should have a brandId and companyId fields, because this query should be part of Domain Repository. This is currently not possible, so this is the only approach to get Brand Domains.